docs/Development/ReleaseNotes.md

```diff
@@ -36,7 +38,12 @@ To check which release of VIC you are running:
 	7. Fixed bug that prevented using the correct local domain grid cells in `cesm_put_data.c`
 	8. Changed reference temperature units from Celsius to Kelvin in `cesm_put_data.c`
 
-	[GH#696](https://github.com/UW-Hydro/VIC/pull/696)
+  ([GH#695](https://github.com/UW-Hydro/VIC/pull/695))
+
+	1. Fix sign for latent heat fluxes passed from VIC to the coupler
+	2. Fix sign for longwave radiation passed from VIC to the coupler
+
+	([GH#696](https://github.com/UW-Hydro/VIC/pull/696))
```

Comment:
there was a merge conflict here that I corrected via github, this does't look quite right though. Can you check/fix this.